index.bs

```diff
@@ -2121,7 +2158,31 @@ Attributes</h4>
         is available to check the readiness of the initial output device.
 
     </ins>
-    </dl>
+</div>
+
+<div class="proposed addition" id="c2567-4">
+    <span class="marker">Proposed Addition
+        <a href="https://github.com/WebAudio/web-audio-api/issues/2567">Issue 2567</a>.
+    </span>
+    Device-related error reporting via AudioContext
+    <div class="amendment-buttons"></div>
+    <ins cite=#2567>
+    : <dfn>onerror</dfn>
+    ::
+        An [=event handler=] for any <dfn event>error</dfn> {{Event}} dispatched from an
```

Comment:
I don't think we want to redefine error... don't you want HTMLMediaElement/error?  

```suggestion
        An [=event handler=] for any <a event for="HTMLMediaElement">error</a> {{Event}} dispatched from an
```

Reply:
Or at least, it should be defined as a AudioContext error.

Reply:
Thanks! Done.

Reply:
This needs a bit more work to define `error` under AudioContext. I'll look for the HTMLMediaElement error spec for examples.